lightning_pose_app/ui/streamlit_video_viewer.py

```diff
@@ -78,77 +78,85 @@ def _render_streamlit_fn(state: AppState):
     model_dir = abspath(os.path.join(proj_dir, MODELS_DIR))
 
     # Streamlit UI
-    st.header("Visualize Model Predictions")
-
-    with st.sidebar:
-        st.write("Select the model and then the associated video you wish to inspect")
-        trained_models = get_models(model_dir)  # create a list of all models
-        selected_model = st.selectbox("**Step 1:** Select a model", trained_models)
-        st.divider()
-        # Labeled videos
-        mp4_files = list_labeled_mp4_files(model_dir, selected_model)
-        selected_video = st.selectbox(
-            "**Step 2:** Select a video", mp4_files, format_func=extract_video_name,
-        )
-        st.divider()
-        # Add a section to download results files
-        video_name = extract_video_name(selected_video)
-        results_files = list_results_files(video_name, model_dir, selected_model)
-
-        if results_files:
-            selected_result_files = st.multiselect(
-                "**Step 3:** Select results files to download", list(results_files.keys())
+    if os.path.exists(model_dir):
+        st.header("Visualize Model Predictions")
+        with st.sidebar:
+            st.write("Select the model and then the associated video you wish to inspect")
+            trained_models = get_models(model_dir)  # create a list of all models
```

Comment:
see file-level comment about replacing this with the `find_models` function from the project backend